app/lib/dttp/entity/degree_qualification.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Dttp
+  module Entity
+    class DegreeQualification
+      attr_reader :degree, :contact_change_set_id, :placement_assignment_change_set_id, :params
+
+      def initialize(degree, contact_change_set_id, placement_assignment_change_set_id)
+        @degree = degree
+        @contact_change_set_id = contact_change_set_id
+        @placement_assignment_change_set_id = placement_assignment_change_set_id
+        @params = build_parms
+      end
+
+      def to_json(*_args)
+        params.to_json
+      end
+
+    private
+
+      def build_parms
+        {
+          "dfe_ContactId@odata.bind" => "$#{contact_change_set_id}",
+          "dfe_TrainingRecordId@odata.bind" => "$#{placement_assignment_change_set_id}",
+          "dfe_ClassofDegreeId@odata.bind" => "/dfe_classofdegrees(#{Entity.degree_class_id(degree.grade)})",
```

Comment:
Harking back to the 'class methods' comment, we could do away with the `Entity` reference here (and all the other places) if they were instance methods. I think these are only called within `Entity`-ses as far as I can see.